add guidelines

deepend-tildeclub opened the pull request:
please take a look.. I want everyone to be involved in curating this so it works for everyone.  I will post the link to this pull request looking for some conversation on it.

Thanks

sneak commented:
Feedback:

- amend item 9 to discourage one-line posts.  "short and to the point" is good, dozens of one-line replies are not.

- strike 10.  "correctly-spelled" is meaningless as language is constantly evolving and this is also somewhat rude to non-native speakers.  as long as the message can be understood, it's good.  not everyone can spel rull gud.

- strike 13.  "real name" policies are bullshit that discriminate against those that need privacy.

- strike 15, or clarify it.  "copyrighted material" as written includes every email you send, as you hold copyright to it.

- amend 16 to include all forms of PII - don't send someone else's PII without their express permission and consent.

- strike 20.  informal community groups are a great way of spreading the word about jobs relevant to the group's members, especially in technical communities.

- strike 22.  you can discuss politics and religion just fine if you follow rules 1 and 2

- strike 23.  humor (especially the misspelled kind) is great.

ramonsnir commented:
Agreed on almost everything! A few opinions: 

> * strike 15, or clarify it.  "copyrighted material" as written includes every email you send, as you hold copyright to it.
"Copyrighted material not owned by you and not licensed to you for redistribution"? The issue here is liability for the mailing list server, so essentially don't share things you can't legally share.

> * strike 20.  informal community groups are a great way of spreading the word about jobs relevant to the group's members, especially in technical communities.
I left several communities because they became job boards by non-participating members. I'd be OK with a rule against posting jobs. 

> * strike 22.  you can discuss politics and religion just fine if you follow rules 1 and 2
I think there should be some distinction here between mildly relevant politics ("Is trademark law fair?", "Does Islam allow NFT resale proceeds to go to the original owner") and irrelevant topics ("Were the elections in Ecuador fair?", "Who was the best apostle?").

deepend-tildeclub commented:
updated some stuff based on your suggestions.  I left the politics and religion one for now.. Need to either leave it or find better wording for it.

sneak commented:
I'll take that as an invitation to suggest replacement copy:

> No discussions about politics or religion, unless you are 100% prepared to abide by rules 1 and 2, and are unlikely to induce others to do so either in a flamewar.  If in doubt, avoid the topic.  *Your* post may be ok, but the ensuing flamewar you cause consumes the time of every single list member.

sneak commented:
Now for additional ones (because these are guidelines and not rules):

* steal ["Assume good faith" from wikipedia](https://en.wikipedia.org/wiki/Wikipedia:Assume_good_faith)
* steal these from the [hacker news guidelines](https://news.ycombinator.com/newsguidelines.html):
    * "Please don't use the mailing list for political or ideological battle."
    * "Please don't pick the most provocative thing in an article or post to complain about in the thread. Find something interesting to respond to instead."
    * "Eschew flamebait. Avoid unrelated controversies and generic tangents."

sneak commented:
The "eschew flamebait" one from HN might be able to replace the no politics/religion one, or combined with it.

deepend-tildeclub commented:
> The "eschew flamebait" one from HN might be able to replace the no politics/religion one, or combined with it.

think that could almost be handled by  number 4.  "debate the issue, do not engage in personal attacks"

deepend-tildeclub commented:
> Now for additional ones (because these are guidelines and not rules):
> 
>     * steal ["Assume good faith" from wikipedia](https://en.wikipedia.org/wiki/Wikipedia:Assume_good_faith)
> 
>     * steal these from the [hacker news guidelines](https://news.ycombinator.com/newsguidelines.html):
>       
>       * "Please don't use the mailing list for political or ideological battle."
>       * "Please don't pick the most provocative thing in an article or post to complain about in the thread. Find something interesting to respond to instead."
>       * "Eschew flamebait. Avoid unrelated controversies and generic tangents."

added the assume good faith.   Added the political or ideological battle one as well (replaced the old political/religious one with this one)

sneak commented:
I think those are great suggestions to put in the doc.  The idea of an extremely low-traffic list having a big list of rules is indeed a bit silly, but these aren't custom-tailored to us, and once they're finalized I intend to adopt the document whole cloth for other projects, and others can do so as well.

deepend-tildeclub commented:
> I wonder though, can we not have guidelines and collectively agree to let bad threads die or go off-list? It’s not like we have a lot of traffic anyways. Just a thought.

Not having any sort of guidelines makes moderation inconsistent and a nightmare.

deepend-tildeclub commented:
> I think those are great suggestions to put in the doc. The idea of an extremely low-traffic list having a big list of rules is indeed a bit silly, but these aren't custom-tailored to us, and once they're finalized I intend to adopt the document whole cloth for other projects, and others can do so as well.

I'd like to see the list grow.  So having a good base to start from is probably a good idea.  And yes it was very much not tailored to this community as it was an example I found online posted unedited.   Just seen it as a good starting off point.

michaelcoyote commented:
I say start with minimal guidelines and iterate as needed.  

I would also suggest that "eschew flamebait" be combined with "assume good faith" along the lines of "act in good faith and assume good faith in others"

michaelcoyote commented:
I would say that posting guidelines for the lists should be basic guidelines.  I agree with you though that there needs to be something and it should be clear. 

On the other hand it might be good if there was a more overarching Code of Conduct to handle the kinds of behaviors on the site as a whole we as a community won't allow.  Rustlang has [a pretty clear example](https://www.rust-lang.org/policies/code-of-conduct).   I would probably take this to the list though...

deepend-tildeclub commented:
> On the other hand it might be good if there was a more overarching Code of Conduct to handle the kinds of behaviors on the site as a whole we as a community won't allow.  Rustlang has [a pretty clear example](https://www.rust-lang.org/policies/code-of-conduct).   I would probably take this to the list though...


We do have a code of conduct. But the CoC isn't all that clear on how it would apply to the mailing list.  Which is why I decided something more specific to the mailing list would be a good approach.

Thanks

Spaceghost commented:
It makes sense to have some set of rules, but it feels distinctly big-net to have a line in the sand instead of expecting earnest and considerate communication. However, I don’t really moderate even IRC anymore, so the grain of sand with my thought it particularly small.

I think a list of “We don’t do this here” is a lot more effective than saying how to be, if that makes sense. Phrasing things in the “No” would be a clear and simple list that would be easier to add to on all sides, and we could probably skip the normal set of “Don’t be a net-negative contributor” rules.
